catalyst_voices/lib/widgets/modals/voices_upload_file_dialog.dart

```diff
@@ -253,14 +253,14 @@ class _UploadContainerState extends State<_UploadContainer> {
           color: Theme.of(context).colors.iconsPrimary!,
           child: Stack(
             children: [
-              DropzoneView(
-                operation: DragOperation.copy,
-                cursor: CursorType.grab,
-                onCreated: (DropzoneViewController ctrl) => setState(() {
-                  _dropzoneController = ctrl;
-                }),
-                onDrop: (ev) async {
-                  if (ev is web.File) {
+              if (kIsWeb)
```

Comment:
can you add a comment explaining why we have that check?

Reply:
done